### ISSUE TITLE
Update headings for multi level models 1

### ISSUE THREAD

**Issue by restrellado** [data-edu/data-science-in-education#189]

**Comment by jrosen48** [comment]
Walkthrough Headers (from our Walkthrough Organization G-Doc, with attribution to https://emilyriederer.netlify.com/post/rmarkdown-driven-development/

```{r}
# Walkthrough 0#: [Type of Analysis] with [Type of Data]

## Introduction: quick summary of the problem to solve/opportunity to attain

### Background: why are we highlighting this? What is the purpose, implications
### Data Source(s): quick su	mmary of what the data is, where did it come from
### Methods: quick summary of the methods that we’ll use in the analysis

## Functions

## Load Packages: List of ALL packages used in the analysis, message = F (but no extra packages)

## Import Data: Load ALL data to be used in the analysis, use descriptive names

## View Data

## Process Data

## Analysis

## Results

## Conclusion
```

**Comment by jrosen48** [comment]
is applicable to #190, #191, #192, #193, and #194 as well

**Comment by jrosen48** [comment]
this is addressed in #233

**Comment by jrosen48** [comment]
I had vocabulary in here cc/ @ivelasq @kierisi. Sorry all, was supposed to have at least started this. I will do for #192 and #194 at least.
